Classes/Service/TheodiaOrg.php

```diff
@@ -436,4 +447,21 @@ private function numberToRomanRepresentation(int $number): string
         }
         return $returnValue;
     }
+
+    /**
+     * Returns the HTTP header to sign the GraphQL query with the configured key
+     */
+    private function sign(Site $site, string $payload): string
+    {
+        $key = $site->getConfiguration()['tx_theodia_api_key'] ?? null;
+        if (!$key) {
+            throw new \RuntimeException("You must configure a theodia API key", 1758789702);
```

Comment:
That's not the way to go in a first step as it will crash every website after an update. We should have a deprecation period, allowing the call to still be made without the key, but having a deprecation message in the logs (as I just wrote in my last comment). 1-2 months after the next release, we'll do another release dropping the deprecation and having that exception instead.

Reply:
done, a deprecation is thrown until December, then a exception will break the website.